src/index.js

```diff
@@ -95,7 +96,9 @@ function startFloat(e) {
 }
 
 function stopFloat(e) {
-  let block_uuid_stop = (e.uuid == undefined) ? e.dataset.heliumId : e.uuid;
+  let block_uuid_stop = (e?.uuid == undefined) ? e : e.uuid;
+  // e.dataset.heliumId was not yet used
+  // let block_uuid_stop = (e.uuid == undefined) ? e.dataset.heliumId : e.uuid;
```

Comment:
`e.dataset.heliumId` is used to identify the block that the balloon icon is attached to - by not using it, the block still floats after clicking on the balloon